MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -18,14 +18,23 @@ final class FeedViewModel {
 		Localized.Feed.title
 	}
 	
+	var errorConnectionrMessage: String {
+		Localized.Feed.errorMessage
+	}
+	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStatus: Observer<Bool>?
```

`onErrorStatus: Observer< Bool > ` could be `onErrorStateChange: Observer<String?>`, so the ViewModel decides which message to show and when to show/hide errors (instead of the view controller). It's often easier to manage state when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed. And when there's an error: `onErrorStateChange(Localized.Feed.errorMessage)`.